test/e2e/egressip.go

```diff
@@ -1591,4 +1622,502 @@ spec:
+
+		ginkgo.By("11. Update the unselected pod to be selected by the Egress IP")
+		pod2 = getPod(f, pod2Name)
+		pod2.Labels = podEgressLabel
+		updatePod(f, pod2)
+
+		ginkgo.By("12. Check connectivity from both pods to an external \"node\" hosted on the OVN managed network " +
+			"and verify that the expected IP")
+		err = wait.PollImmediate(retryInterval, retryTimeout, targetExternalContainerAndTest(targetNode, pod1Name, podNamespace.Name,
+			true, []string{egressIPOVNManaged}))
+		framework.ExpectNoError(err, "Step 12. Check connectivity from pod (%s/%s) to an external container attached to "+
+			"a network that is OVN managed and verify that the src IP is the expected egress IP, failed, err: %v", podNamespace.Name, pod1Name, err)
+		err = wait.PollImmediate(retryInterval, retryTimeout, targetExternalContainerAndTest(targetNode, pod2Name, podNamespace.Name,
+			true, []string{egressIPOVNManaged}))
+		framework.ExpectNoError(err, "Step 12. Check connectivity from pod (%s/%s) to an external container attached to "+
+			"a network that is OVN managed and verify that the src IP is the expected egress IP, failed, err: %v", podNamespace.Name, pod2Name, err)
+
+		ginkgo.By("13. Check connectivity from both pods to an external \"node\" hosted on non-OVN managed network " +
+			"and verify that the expected IP")
+		err = wait.PollImmediate(retryInterval, retryTimeout, targetExternalContainerAndTest(targetSecondaryNode, pod1Name, podNamespace.Name,
+			true, []string{egressIPNonOVNManaged}))
+		framework.ExpectNoError(err, "Step 13. Check connectivity from pod (%s/%s) to an external container attached to "+
+			"a network that isn't OVN managed and verify that the src IP is the expected egress IP, failed, err: %v", podNamespace.Name, pod1Name, err)
+		err = wait.PollImmediate(retryInterval, retryTimeout, targetExternalContainerAndTest(targetSecondaryNode, pod2Name, podNamespace.Name,
+			true, []string{egressIPNonOVNManaged}))
+		framework.ExpectNoError(err, "Step 13. Check connectivity from pod (%s/%s) to an external container attached to "+
+			"a network that isn't OVN managed and verify that the src IP is the expected egress IP, failed, err: %v", podNamespace.Name, pod2Name, err)
+
+		ginkgo.By("14. Set the node hosting the OVN managed egress IP as unavailable")
+		var nodeNameHostingOVNManagedEIP, nodeNameHostingNonOVNManagedEIP string
+		for _, status := range statuses {
+			if status.EgressIP == egressIPOVNManaged {
+				nodeNameHostingOVNManagedEIP = status.Node
+			} else if status.EgressIP == egressIPNonOVNManaged {
+				nodeNameHostingNonOVNManagedEIP = status.Node
+			}
+		}
+		gomega.Expect(nodeNameHostingOVNManagedEIP).ShouldNot(gomega.BeEmpty())
+		gomega.Expect(nodeNameHostingNonOVNManagedEIP).ShouldNot(gomega.BeEmpty())
+		egressNodeAvailabilityHandler.Disable(nodeNameHostingOVNManagedEIP)
+
+		ginkgo.By("15. Check that the status is of length one")
+		statuses = verifyEgressIPStatusLengthEquals(1, nil)
```

Comment:
should we check that the status has the egress ip that we expect here and in similar other steps?

Reply:
Done